e-commerce-project/src/app/components/login-page/login-page.component.ts

```diff
@@ -1,4 +1,5 @@
 import { Component, OnInit } from '@angular/core';
+import {FormBuilder, FormGroup, Validators} from '@angular/forms';
```

Comment:
There are few classes we import from @angular/forms, which sounds the same but they have slightly different purpose which you don't need to remember (just take a look and practice a bit with them):
* FormBuilder - creates instance of  
* FormGroup - creates instance of  binds in html template 
* Validators - helps with validation

Note: do not worry if you cannot understand from first glance it takes time.
The best way to figure it out is to practice a bit.
Look at examples in the documentation https://angular.io/guide/reactive-forms#step-3-generating-form-controls